### DISCUSSION TITLE
Support the Mobile Native Foundation

### DISCUSSION THREAD

**Discussion by carloshpdoc** [MobileNativeFoundation/discussions#66] [General]
Hi, guys!

I need help.  How the [enterprise](globo.com) I working can support the Mobile Native Foundation?

Thanks for the help.

**Comment by keith** [comment]
Thanks for your interest! I think the biggest thing we're hoping to do right now is build our community through these discussions. We're also hoping that from these discussions we can identify some places where working on projects as part of the foundation makes sense. So please jump in wherever you're interested to and encourage others to do the same!

**Comment by 8vqjx80b** [comment]
Mobile Native Foundationin tukeminen on järkevää, jos haluatte vaikuttaa mobiilialan standardeihin. Paras tapa on osallistua työryhmiin, joissa ratkotaan yhteisiä ongelmia, kuten CI-skaalausta. Jos yrityksellänne [huippu-casino.net](https://huippu-casino.net/) on sisäisiä työkaluja, niiden avaaminen avoimeksi lähdekoodiksi on myös todella arvokasta. Kysykää heidän sivuiltaan yritysjäsenyydestä.
